### ISSUE TITLE
System does not build with catppuccin nix,  "tofi.settings" does not exist

### ISSUE THREAD

**Issue by Haskellered** [catppuccin/nix#154]
Nixos 23.11
```
       error: The option `home-manager.users.haskeller.programs.tofi.settings' does not exist. Definition values:
       - In `/nix/store/4x11z78x84bvvx740jx16d312bvz82pv-source/flake.nix':
           {
             _type = "if";
             condition = false;
             content = {
               background-color = "#24273a";
           ...
```

**Comment by Lichthagel** [comment]
You are probably running an outdated version of home-manager. The tofi module was added 10 days ago: https://github.com/nix-community/home-manager/commit/8ff7bb3f4d82a05a52d5242ec454bba14f5d6cc6.

**Comment by cgoetzke** [comment]
> these modules are based on the master branch of hm, and we don't support the release branches just yet

Does HM master track nixpkgs-unstable? For those of us tracking the latest nixpkgs release (23.11 right now), what does this mean? Will this shake out when 24.05 drops?

**Comment by cgoetzke** [comment]
For what it's worth, my hacky workaround is to pin catppuccin-nix for the time being:

```
{
  inputs = {

    ...

    # Temporarily pin catppuccin, since they don't support tracking releases
    # currently, and newer commits break home-manager 23.11
    #catppuccin.url = "github:catppuccin/nix";
    catppuccin.url = "github:catppuccin/nix/a48e70a31616cb63e4794fd3465bff1835cc4246";
  
  ...

}
```

I'll probably try unpinning once I upgrade to 24.05.

Also, that catppuccin-nix commit was chosen somewhat randomly, but it does allow me to update / `nixos-rebuild` my system.

**Comment by harryfinbow** [comment]
For anyone who was getting this error while on unstable running `nix flake update`, which bumps `home-manager` to use the latest commit (which contains the new tofi module), fixed this for me.

**Comment by rgroemmer** [comment]
using `nixos-unstable` and getting:
```       
error: The option `home-manager.users.rap.wayland.windowManager.hyprland.settings' does not exist. Definition values:
       - In `/nix/store/rpfzqsa6kx5a9ch2vn2j5jylx42ijsl8-source/home-manager':
 ```

**Comment by getchoo** [comment]
this should be fixed as of aef567291242b03e141ba68375c2ff75ea8ff676. i would recommend all of those affected to run `nix flake update`

keeping this pinned for a bit in case anyone else comes across this
